sdk/appcenter/src/main/java/com/microsoft/appcenter/http/HttpException.java

```diff
@@ -74,52 +39,26 @@ private static String getDetailMessage(int status, @NonNull String payload) {
     }
 
     /**
-     * Get the HTTP status code.
-     *
-     * @return HTTP status code.
-     */
-    public int getStatusCode() {
-        return statusCode;
-    }
-
-    /**
-     * Get the HTTP payload (response body).
-     *
-     * @return HTTP payload. Can be empty string.
-     */
-    @NonNull
-    public String getPayload() {
-        return payload;
-    }
-
-    /**
-     * Get the HTTP headers (response headers).
+     * Get the HTTP response.
      *
-     * @return HTTP headers.
+     * @return HTTP response.
      */
-    @NonNull
-    public Map<String, String> getHeaders() {
-        return headers;
+    public HttpResponse getHttpResponse() {
+        return mHttpResponse;
     }
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
+        if (this == o) return true;
```

Comment:
I assume this method is generated by Android Studio, interesting that the generated code omits the curly brackets. Not a problem because we are supposed to always auto-generate this method.